sdk/typescript/README.md

```diff
@@ -33,6 +33,12 @@ cd <your project directory>
 npm link @mysten/sui.js
 ```
 
+If you wish to rebuild all dependencies of the TypeScript SDK, or if you're encountering issues with the `@mysten/bcs` module not being found, you can re-build the SDK module and all of it's local dependencies using the following command:
```

Comment:
For @randall-Mysten, please correct the contraction-for-possessive issue here:
https://www.grammarly.com/blog/contractions/